magento/pwa-studio
# Pwa 1471

## Description
PR to add functionality to parse media query CSS rules that are set up in pageBuilder and apply their respective styles to the following content types:

- Row
- Column
- Slider
- Banner
- Tabs
- Tab Item

## Related Issue







Closes PWA-1471

## Acceptance





PWA storefront should be able to render styles set with media query breakpoints configured in Page Builder for the content types mentioned above.

### Verification Stakeholders



### Specification




## Verification Steps






#### Test scenario(s) for direct fix/feature





1. Verify desktop and device specific breakpoints are configured in Page Builder
2. Verify the PWA storefront renders media query breakpoints for desktop and devices for multiple Page Builder content types: Rows, Columns, Tabs, Tab Items, Banners, Sliders, and Slides
3. Verify that all existing cypress snapshots tests are passing
4. Verify that all media query cypress snapshots tests are passing
#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?
No




#### Any ad-hoc/edge case scenarios that need to be considered?
No




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist
